worlds/lingo/player_logic.py

```diff
@@ -153,17 +162,29 @@ def __init__(self, world: "LingoWorld"):
         victory_condition = world.options.victory_condition
         early_color_hallways = world.options.early_color_hallways
 
-        if location_checks == LocationChecks.option_reduced and door_shuffle != ShuffleDoors.option_none:
-            raise OptionError("You cannot have reduced location checks when door shuffle is on, because there would not"
-                              " be enough locations for all of the door items.")
+        if location_checks == LocationChecks.option_reduced:
+            if door_shuffle == ShuffleDoors.option_doors:
+                raise OptionError("You cannot have reduced location checks when door shuffle is on, because there "
```

Comment:
These warning should include the player name

Reply:
Done!